### ISSUE TITLE
Global header and footer

### ISSUE THREAD

**Issue by kurt-leingang** [beer-gang/beer-gang#29]
While bar and brewery views are listed individually, both will actually be one global header and footer which dynamically modifies the menu options based on whether the user is a bar or brewery.

- [ ] ~[Bar Global Header](https://i.gyazo.com/9b94fc24af7dfdb81232015a81e5e7e0.png)~

     - Company logo
     - Menu bar
     - (do not include page title "Breweries", this was left in to show logo and menu positioning)
Example HTML for bar header:
```html
<div class="dropdown">
  <a class="btn btn-light dropdown-toggle float-right" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Menu
  </a>

  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink" style="">
    <a class="dropdown-item" href="https://beer-gang-production.herokuapp.com/pour/orders">Order Management</a>
    <a class="dropdown-item" href="https://beer-gang-production.herokuapp.com/breweries">Brewery Select</a>
    <a class="dropdown-item" href="https://beer-gang-production.herokuapp.com/bars/1/edit">Account Details</a>
<a class="dropdown-item" href="#">Logout</a>
  </div>
</div>
   <div class="text-center" style="
    margin-top: -40px;
    margin-bottom: 20px;
">
        <img src="https://i.gyazo.com/b3a912a245f2317f6da62598787db66f.png" style="
    max-width: 300px;
    margin-left: 60px; <!-- This image displays slightly to the left of center due to the menu button. I manually updated margin-left to align in the photo, but there's definitely a better way to do this permanently -->
">
      </div>
```

- [ ]  ~[Brewery Global Header](https://i.gyazo.com/b50b0cbff1c18a08c5c027df366e8749.png)~
     - Company logo
     - Menu bar
     - (do not include page title "Orders", this was left in to show logo and menu positioning)

Example HTML for brewery header:
```html
<a class="btn btn-light dropdown-toggle float-right" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Menu
  </a>

  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink" style="">
    <a class="dropdown-item" href="https://beer-gang-production.herokuapp.com/orders">Orders</a>
    <a class="dropdown-item" href="https://beer-gang-production.herokuapp.com/beers">Manage Beers</a>
    <a class="dropdown-item" href="https://beer-gang-production.herokuapp.com/breweries/1/edit">Edit Brewery</a>
<a class="dropdown-item" href="#">Logout</a>
  </div>
</div>
    
    <div class="text-center" style="
    margin-top: -40px;
    margin-bottom: 20px;
">
        <img src="https://i.gyazo.com/b3a912a245f2317f6da62598787db66f.png" style="
    max-width: 300px;
    margin-left: 60px; <!-- This image displays slightly to the left of center due to the menu button. I manually updated margin-left to align in the photo, but there's definitely a better way to do this permanently -->
">
      </div> 
```

- [ ] [Global Footer](https://i.gyazo.com/78961b690abeffa5465668b85c67dce2.png)
    - [x] link Terms of Use to /terms

**Comment by kurt-leingang** [comment]
@zabrador looks like the global footer [doesn't take scrolling into account](https://i.gyazo.com/38e819ba500570d7615305a7d9923f4b.png) FYI.

**Comment by kurt-leingang** [comment]
Any reason why the header has so much [margin from the top of the page](https://i.gyazo.com/16a39b15809134a71f9da519de178672.png)?

**Comment by kurt-leingang** [comment]
I know it's also a bit of an edge case, but if you [squash the page on desktop the image doesn't center](https://i.gyazo.com/668ee6d03db910fd93d59b1853360b33.png). Since this doesn't happen on mobile we probably can make note of it and ignore it for now, but wanted to call it out for the sake of perfectionism.

**Comment by kurt-leingang** [comment]
- [ ] I recommend adding margin to the bottom of the header to add some space between the logo and whatever comes next on the page.

**Comment by kurt-leingang** [comment]
Could you please scale back the logo and menu button to match the left-right margins of the table? [Here's how it is now](https://i.gyazo.com/8c2238d46794f2a9f659a43d754fd58f.png) and [here's what I think would look a bit cleaner](https://i.gyazo.com/529991ec79a0698b0b284b018b06a307.png).

**Comment by zabrador** [comment]
> @zabrador looks like the global footer doesn't take scrolling into account FYI.

Yeah I was noticing that after I added it, gonna have to spend some time playing around with the absolute positioning.

> Any reason why the header has so much margin at the top of the page?

Two notification paragraphs for error and success notifications. Something on my to-do list but not a trivial solve, so I haven't prioritized it.

> I know it's also a bit of an edge case, but if you squash the page on desktop the image doesn't center. Since this doesn't happen on mobile we probably can make note of it and ignore it for now, but wanted to call it out for the sake of perfectionism.

It actually IS centered, according to the way the grid layout responds. This is the same issue as the login page where things are responding poorly because of how large the image is. We should probably just hide it on mobile until we have the new replacement.

> I recommend adding margin to the bottom of the header to add some space between the logo and whatever comes next on the page.

Is this related to the overlap on login? Let's wait to see how things play out with a new header image and header text before making one-off adjustments, those will very quickly become difficult to keep track of.

> Could you please scale back the logo and menu button to match the left-right margins of the table? Here's how it is now and here's what I think would look a bit cleaner.

Certainly, wanna just send over another image in whatever size you want it to be?

---

As a general FYI, I haven't been paying almost any attention to mobile because of that root responsiveness issue we noticed (where it adds the huge margins back again) and because the more immediate urgency was for breweries (who wouldn't normally use mobile). We may want to spend time on that soon since that initial "deadline" has finished.
